hardhat.config.ts

```diff
@@ -55,27 +56,27 @@ task('publish', "verify and publish contracts on etherscan")
 
     try{
         //InflationController
-        await hre.run("verify:verify", {
-            address: deployment.InflationController.innerAddress,
-            constructorArguments: [],
-        });
+        // await hre.run("verify:verify", {
```

Comment:
[Question] why comment out these codes?

Reply:
it is used for verify contract on polygonscan last time